packages/react-native/example/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@compassion-design-system/react-native-example",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
```

Comment:
I ran `npm i` to try and run the example and got the error below. I’m sure things are fine, but would love to have a runnable example while we work on building out Storybook for RN. This was a clean install, having run `npm i` from the monorepo root (after specifying Node v18), and then running it again here. I could just be doing something wrong. 😅 

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: react-scripts@5.0.1
npm ERR! Found: typescript@5.3.3
npm ERR! node_modules/typescript
npm ERR!   dev typescript@”^5.0.2” from the root project
npm ERR!   typescript@”^4.6.4 || ^5.2.2” from @commitlint/load@17.8.1
npm ERR!   node_modules/@commitlint/load
npm ERR!     @commitlint/load@”^17.8.1” from @commitlint/cli@17.8.1
npm ERR!     node_modules/@commitlint/cli
npm ERR!       @commitlint/cli@”^17.8.1” from commitlint@17.8.1
npm ERR!       node_modules/commitlint
npm ERR!         dev commitlint@”^17.0.2” from the root project
npm ERR!   5 more (cosmiconfig, cosmiconfig-typescript-loader, …)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional typescript@”^3.2.1 || ^4” from react-scripts@5.0.1
npm ERR! node_modules/react-scripts
npm ERR!   react-scripts@”latest” from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: typescript@4.9.5
npm ERR! node_modules/typescript
npm ERR!   peerOptional typescript@”^3.2.1 || ^4” from react-scripts@5.0.1
npm ERR!   node_modules/react-scripts
npm ERR!     react-scripts@”latest” from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with –force, or –legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```